docs/docs/tools.md

```diff
@@ -0,0 +1,38 @@
+
+# Tools
+
+## `muspinsim-gen`
+
+`muspinsim-gen` is a command line tool made to help construct an input file given a cell structure file as input. At the moment it can take an input structure (defined in a file supported by `ase` e.g. a .cell or .cif), find a requested number of nearest neighbour atoms to the muon, and then output the muspinsim config defining these atoms and the dipolar interactions with the muon. It can also be linked to a file containing output defining EFG tensors from `GIPAW` in order to include quadrupole interactions.
```

Comment:
```suggestion
`muspinsim-gen` is a command line tool made to help construct an input file given a cell structure file as input. At the moment it can take an input structure (defined in a file supported by `ase` e.g. a .cell or .cif), find a requested number of nearest neighbour atoms to the muon, and then output the muspinsim config defining these atoms and the dipolar interactions with the muon. It can also (optionally) take a file containing output defining EFG tensors from `GIPAW` in order to include quadrupole interactions.
```
Just for clarity and avoid any confusion with actally linking file locations. We describe `tak[ing] an input structure...` so think it's best to use the same langauge throughout.